### PR TITLE
Issue/home page

### PR DESCRIPTION
**Pull request by yuvalron**
image

### CODE REVIEW

**Comment by yuvalron** [comment]
image

**Comment by yuvalron** [comment]
image

File: src/app/ui/home/home.component.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<div class="container">
+  <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
```
Comment (menhel) [inline src/app/ui/home/home.component.html]: Watch the id: `id="carouselExampleIndicators"`. Change it to something the describes the actual element.

Comment (menhel) [reply src/app/ui/home/home.component.html]: Make sure you change the id in all referenced locations too. For example, in lines 4,5,6 and many more.

File: src/app/ui/home/home.component.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+      <span class="sr-only">Previous</span>
+    </a>
+    <a class="carousel-control-next" href="#carouselExampleIndicators" role="button" data-slide="next">
+      <span class="carousel-control-next-icon" aria-hidden="true"></span>
+      <span class="sr-only">Next</span>
+    </a>
+  </div>
+</div>
+<div class="row m-4 justify-content-center">
```
Comment (menhel) [inline src/app/ui/home/home.component.html]: We don't know yet what will be on the homepage, so why adding cards (which are not even links) to it, as if we can explain the action the suppose to symbol?
I think these cards should be removed.

File: src/app/ui/home/home.component.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<div class="container">
+  <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
+    <ol class="carousel-indicators">
+      <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
+      <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
+      <li data-target="#carouselExampleIndicators" data-slide-to="2"></li>
+    </ol>
+    <div class="carousel-inner shadow-lg bg-white rounded">
```
Comment (menhel) [inline src/app/ui/home/home.component.html]: Regarding the names of the images, please create a directory inside `assets/images` which is called homepage and add the images there. Also, rename each image to describe it's content, and not 1,2,3...

File: src/app/ui/home/home.component.html (hunk 2 above)
Comment (yuvalron) [reply src/app/ui/home/home.component.html]: removed.

**Pull request merged**